.github/workflows/release-drafter.yml

```diff
@@ -2,9 +2,13 @@ name: Release Drafter
 
 on:
   workflow_dispatch:
+  #This job will be triggered when changes are pushed to the master branch, and it will also be triggered when a branch is created with a name that starts with 'refs/heads/releases-'
   push:
     branches:
-      - master
+      push:
+        branches:
+          - master
+          - 'release-**'
```

Comment:
so we call it a rule... release branches needs to start with "release-"

Reply:
I think having a set rule for branch name would be a better standard practise to have. Please let me know if you feel otherwise or want the branch-name to start with something else.